synapse/rest/client/v2_alpha/register.py

```diff
@@ -377,6 +378,41 @@ async def on_GET(self, request):
             return 200, {"available": True}
 
 
+class RegistrationTokenValidityRestServlet(RestServlet):
+    PATTERNS = client_patterns(
+        f"/org.matrix.msc3231/register/{LoginType.REGISTRATION_TOKEN}/validity",
+        unstable=True,
+    )
+
+    def __init__(self, hs):
```

Comment:
Also, could you add a docstring with an example request/response to this servlet as well?

Reply:
Sure